### PR TITLE
Fix go package name

### LINKED ISSUE #7: create semvar tag and update the import "codahale/hdrhistogram" to "hdrhistogram"?
A semvar tag is helpful for dependency management tool like `go module`.
And because the test files import `codahale/hdrhistogram` instead of `hdrhistogram`, `go module` will consider `codahale/hdrhistogram` as a dependency of this project. Actually, use the `hdrhistogram` without namespace is enough.

### PR DESCRIPTION
**Pull request by zoidyzoidzoid**
Closes https://github.com/zoidbergwill/hdrhistogram/issues/7

@spacewander, you mind taking a look?

How will this work with `go get`?

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module hdrhistogram
```
Comment (spacewander) [inline go.mod]: The module name seems incorrect. While downloading with `go get github.com/zoidbergwill/hdrhistogram@zoid/issue-7`, I got an error: `parsing go.mod: unexpected module path "hdrhistogram"`.

The module name should be a full uri like https://github.com/caddyserver/caddy/blob/master/go.mod#L1

Comment (zoidyzoidzoid) [reply go.mod]: Yeah, when it generated one it used "github.com/zoidbergwill/hdrhistogram"

**Comment by zoidyzoidzoid** [comment]
Maybe I should try move it to the hdrhistogram org instead of using my personal account?

File: go.mod (hunk 1 above)
Comment (spacewander) [reply go.mod]: @zoidbergwill 
Now the `go get` works for me.

**Pull request merged**